app/assets/stylesheets/landing_page/sections/hero.scss

```diff
@@ -239,6 +243,14 @@
 
   padding: 18px 36px;
 
+  // Hover color is used to indicate tab focus.
+  // The hover color is set in the layout
+  outline: none;
```

Comment:
Are the `:focus` styles coming from somewhere else? An element can be in focus, but not in hover state.